cli/client/http.go

```diff
@@ -5,68 +5,85 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
-	"github.com/mesosphere/dcos-commons/cli/config"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
 	"path"
 	"strings"
+
+	"github.com/mesosphere/dcos-commons/cli/config"
 )
 
-func HTTPGet(urlPath string) *http.Response {
-	return CheckHTTPResponse(HTTPQuery(CreateHTTPRequest("GET", urlPath)))
+func HTTPServiceGet(urlPath string) *http.Response {
```

Comment:
IMO I think it'd be fine to leave the current Service ones with the original "default" names. The Cosmos queries in comparison are still a bit of a special case.